### PR TITLE
Remove parallel verification

### LINKED ISSUE #1420: Parallel verification issues
**Describe the bug**
While reviewing the bug #1410 I used multiple threads for the rpc request and I advice that it produce a similar denial of services that is described in #1410

The error was produced here https://github.com/neo-project/neo/blob/fc1a64b77c8a901c048e7b973f204d3b7c9870e3/src/neo/Ledger/Blockchain.cs#L500
image

Because a double spend in Gas native token https://github.com/neo-project/neo/blob/fc1a64b77c8a901c048e7b973f204d3b7c9870e3/src/neo/SmartContract/Native/Tokens/Nep5Token.cs#L100

image

**To Reproduce**
1. Create a 1 node network
2. install rpc module
3. open wallet by rpc
4. send multiple sends with at least 5 threads

```json
{
  "jsonrpc": "2.0",
  "method": "sendtoaddress",
  "params": ["0x8c23f196d8a1bfd103a9dcb1f9ccf0c611377d3b", "NNa1Fgc82Qoh65TJfViqC92W8Z8AWL9YKA",0.00000001],
  "id": 1
}
```

**Expected behavior**
Consensus nodes are expected to handle massive txs.

**Additional context**
Possible fix https://github.com/neo-project/neo/pull/1417

### PR DESCRIPTION
**Pull request by shargon**
Close #1420

Alternative of https://github.com/neo-project/neo/pull/1417

### CODE REVIEW

**Comment by shargon** [comment]
This PR is for testing the speed between the different solutions

**Comment by cloud8little** [comment]
@shargon  @vncoelho see test result on https://github.com/neo-project/neo/pull/1423#issuecomment-582733348

**Comment by shargon** [comment]
Closed because is slower than #1423 and fix the same thing, thanks @cloud8little !

File: src/neo/Ledger/MemoryPool.cs [hunk 1]
```diff
@@ -416,7 +416,7 @@ internal void InvalidateAllTransactions()
             // Since unverifiedSortedTxPool is ordered in an ascending manner, we take from the end.
             foreach (PoolItem item in unverifiedSortedTxPool.Reverse().Take(count))
             {
-                if (item.Tx.VerifyForEachBlock(snapshot, SendersFeeMonitor.GetSenderFee(item.Tx.Sender)) == RelayResultReason.Succeed)
+                if (item.Tx.Verify(snapshot, SendersFeeMonitor.GetSenderFee(item.Tx.Sender)) == RelayResultReason.Succeed)
```
Comment (erikzhang) [inline src/neo/Ledger/MemoryPool.cs]: We can keep `VerifyForEachBlock()` to improve the performance of mempool.

Comment (shargon) [reply src/neo/Ledger/MemoryPool.cs]: And move `VerifyParallelParts` to `Verify` ?

Comment (erikzhang) [reply src/neo/Ledger/MemoryPool.cs]: Yes.

Comment (shargon) [reply src/neo/Ledger/MemoryPool.cs]: Done

**Comment by erikzhang** [comment]
@cloud8little Need to be test.

**Comment by cloud8little** [comment]
> @cloud8little Need to be test.

ok

**Pull request merged**